nino/endpoints.py

```diff
@@ -57,53 +92,77 @@ def __init__(self, client: AniClient, name: str, page: int, per_page: int):
         self.per_page = per_page
 
     @property
-    def title(self):
+    def title(self) -> Union[dict, list]:
```

Comment:
```suggestion
    def title(self) -> Union[dict, list[dict]]:
```
Feels odd when the docstring says "list of dicts", might as well show that in the annotation too (this suggestion might fail if you don't have the `from __future__ import annotations` at the top of the file

Reply:
this isnt very important tho 🤷🏻